python/ray/util/sgd/v2/trainer.py

```diff
@@ -361,14 +361,17 @@ def latest_checkpoint_dir(self) -> Optional[Path]:
         return self._executor.latest_checkpoint_dir
 
     @property
-    def latest_checkpoint_path(self) -> Optional[Path]:
-        """Path to the latest persisted checkpoint from the latest run.
+    def best_checkpoint_path(self) -> Optional[Path]:
```

Comment:
Do we also want to expose a `best_checkpoint` object the same way we have for `latest_checkpoint`?